scripts/ChampionsRoom.asm

```diff
@@ -268,31 +268,31 @@ GaryVictoryText:
 	text_far _GaryVictoryText
 	text_end
 
-GaryText_76103:
-	text_far _GaryText_76103
+GaryAfterBattleText:
```

Comment:
I feel like instead of using "Gary" even more, we should replace it with "Rival" like Gen 2. The ChampionsRoom and OaksLab script and text files are the only places left that refer to Gary. Elsewhere we already use "Rival", e.g. `OPP_RIVAL1`, `EVENT_BEAT_POKEMON_TOWER_RIVAL`, `MUSIC_MEET_RIVAL`, etc.